### PR TITLE
[11.0][ADD] account_menu_invoice_refund: Backport from 12.0

### PR DESCRIPTION
**Pull request by GerardoGomez**
Backport from 12.0

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
Thank you for the PR. The commit history here is missing, can you add it? In order to do that you just need to be even to version 11.0 and then do the `git format-patch --keep-subject --stdout origin/11.0..origin/12.0 -- account_menu_invoice_refund | git am -3 --keep `thing. Then commit the backport changes

**Comment by GerardoGomez** [comment]
> Thank you for the PR. The commit history here is missing, can you add it? 

Thanks for your comment @AaronHForgeFlow. I was not sure if it was necessary to do it in the backports

File: account_menu_invoice_refund/__init__.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# © 2021 FactorLibre - Gerardo Gomez-Caminero <gerardo.gomez@factorlibre.com>
```
Comment (dreispt) [inline account_menu_invoice_refund/__init__.py]: Please remove copyright claims.
Copyright applies to original work author only, and porting the code is not enough to qualify for authorship.

**Review by dreispt — CHANGES_REQUESTED** [review]

File: account_menu_invoice_refund/__init__.py (hunk 1 above)
Comment (RodrigoBM) [reply account_menu_invoice_refund/__init__.py]: Removed

**Pull request closed without merging**